docs/UserGuide.md

```diff
@@ -15,23 +15,28 @@
 1. Download the latest executable Jar on our [release](https://github.com/reposense/RepoSense/releases/latest).
    * Alternatively, you can compile the executable Jar yourself by following our [build from source guide](Build.md).
 1. Execute it on the OS terminal. <br>
-Usage: `java -jar RepoSense.jar -config CSV_CONFIG_FILE_PATH [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]`
+Usage: `java -jar RepoSense.jar -config CSV_CONFIG_FILE_PATH [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...] [-view REPORT_DIRECTORY]`
```

Comment:
why is `REPORT_DIRECTORY` needed? It should be automatic to save user the trouble.

Reply:
Because the user might choose an external directory. Plus, `-view` will be just used to display the dashboard. No report will be generated, so its necessary for the user to point to the location of the report directory.